website/index.html

```diff
@@ -20,88 +20,90 @@ <h2>
                 </h2>
             </a>
         </header>
-        <div class="flex-all-content">
+        <div class="content">
```

Comment:
Suggestion: Separate css into animations/keyframes, and then the rest in `style.css`

Reply:
7fab555